test/statsCases/import-context-filter/entry.js

```diff
@@ -0,0 +1,21 @@
+async function getTemplate(templateName) {
+	try {
+		let template = await import(
+			/* webpackFilter: "\\.js$" */
+			/* webpackExclude: "\\.noimport\\.js$" */
```

Comment:
It's also possible to write `/* webpackFilter: "\\.js$", webpackExclude: "\\.noimport\\.js$" */`